Scripts/Items/Consumables/EodonPotions.cs

```diff
@@ -802,30 +802,61 @@ public override void Deserialize(GenericReader reader)
         }
     }
 
-    public class BlueCornStock : Item
```

Comment:
Are these on spawners or do they have an internal spawner?  If so, the spawners need to be changed (in the spawner file) as well as changing the name on any existing spawners.  See GiantSerpent.cs for how to do this.  Or, you can just keep the name as BlueCornStock to avoid this.